docs/installation.rst

```diff
@@ -30,7 +30,7 @@ this method by following the steps described below.
 
   ::
 
-    $ wget -q https://github.com/tardis-sn/tardis/blob/master/conda-{platform}-64.lock
+    $ wget -q https://raw.githubusercontent.com/tardis-sn/tardis/master/conda-linux-64.lock
```

Comment:
I think if you're going to change this here you should also change the surrounding documentation to match. As is it's confusing with the lines that come before it.

Reply:
yup you are right~ Thanks for catching that!